client/packages/system/src/Item/DetailView/Tabs/General.tsx

```diff
@@ -31,66 +33,67 @@ export const GeneralTab: FC<GeneralTabProps> = ({}) => {
         <DetailSection title={t('title.details')}>
           <DetailInputWithLabelRow
             label={t('label.name')}
-            inputProps={{ value: data?.name, disabled: isDisabled }}
+            inputProps={{ value: item?.name, disabled: isDisabled }}
           />
           <DetailInputWithLabelRow
             label={t('label.code')}
-            inputProps={{ value: data?.code, disabled: isDisabled }}
+            inputProps={{ value: item?.code, disabled: isDisabled }}
           />
 
           <DetailInputWithLabelRow
             label={t('label.unit')}
-            inputProps={{ value: data?.unitName, disabled: isDisabled }}
+            inputProps={{ value: item?.unitName, disabled: isDisabled }}
           />
 
           <DetailInputWithLabelRow
             label={t('label.strength')}
-            inputProps={{ value: data?.strength, disabled: isDisabled }}
+            inputProps={{ value: item?.strength, disabled: isDisabled }}
           />
           <DetailInputWithLabelRow
             label={t('label.ddd')}
-            inputProps={{ disabled: isDisabled }}
             Input={
               <NumericTextInput
-                value={Number(data?.ddd)}
+                value={Number(item?.ddd)}
                 disabled={isDisabled}
+                fullWidth
               />
             }
-            DisabledInput={<NumericTextDisplay value={data?.ddd} />}
           />
           <DetailInputWithLabelRow
             label={t('label.type')}
-            inputProps={{ value: data?.type, disabled: isDisabled }}
+            inputProps={{ value: item?.type, disabled: isDisabled }}
           />
           <DetailInputWithLabelRow
             label={t('label.doses')}
-            inputProps={{ disabled: isDisabled }}
             Input={
-              <NumericTextInput value={data?.doses} disabled={isDisabled} />
+              <NumericTextInput
```

Comment:
lil side quest to remove the `DisabledInput` all that white space between the label and number was making it hard to match up what number goes with what label..